crates/zed_actions/src/lib.rs

```diff
@@ -185,7 +185,8 @@ pub struct OpenRecent {
 }
 
 impl_actions!(projects, [OpenRecent]);
-actions!(projects, [OpenRemote]);
```

Comment:
We should restore `OpenRemote` using the `action_with_deprecated_aliases` macro.

Reply:
Sorry I won't have the capacity to follow through with this.
Anyone who finds this PR, feel free to pick up.